striim-transform/README.md

```diff
@@ -0,0 +1,79 @@
+# Striim Compatible Transformer
+
+[Striim](https://www.striim.com) is a distributed data integration and intelligence platform that can be used to design, deploy, and run data movement and data streaming pipelines.
+
+This repository contains simple steps to set up a YugabyteDB CDC pipeline with `StriimCompatible` transfomer that transforms the messages being sent to kafka in a format compatible with Striim's [WAEvent](https://www.striim.com/docs/en/postgresql-cdc.html#UUID-bb4c7685-acc1-6f6a-f4a2-afbfecf4febf).
+
+## Setup
+
+To use `StriimCompatible` transform with the connectors the following properties needs to be added to the connector configuration.
+
+```json
+"transforms": "unwrap",
```

Comment:
`unwrap` is just a name we give to the transformer. However, in this case, the name seems a little misleading. Maybe we can rename this to `consistent` or `consistency` or `striim`.

Reply:
Oh okay, done.